.github/workflows/cd.yml

```diff
@@ -49,9 +49,18 @@ jobs:
       - name: Print Environment Information
         run: npx nx report
 
-      - name: Lint, test, and build affected packages
-        run: npx nx affected -t lint,test,build
-        
+      - name: Format affected packages
+        run: nx run-many --target=format --all
```

Comment:
Do we need to run this and line 56 against all, or can we run them against just the affected packages?

Reply:
Good point, i think we can just run against affected - i'll update!